script.js

```diff
@@ -0,0 +1,87 @@
+const chatbot = [
+    {
+        input:  ['hello', 'hi', 'greetings'],
+        output: ['Hello', 'Hey', 'Greetings']
+    },
+    {
+        input:  ['what is your favourite colour?', 'who is your favourite HYF instructor?', 'who is your role model?'],
+        output: ['I am not sure.','There are too many to choose from', 'I like every one']
+    },
+   
+    {
+        input:  ['how are you?', 'how is the weather today?', 'how is Canada doing in the Olympics?'],
+        output: ['Fine', 'Great', 'Not so good']
+    },
+    
+];
+
+
+//print variable
+console.log(chatbot)
+
+//created a function called ‘reply’.
+function reply() {
+  
+//input selected, assigned a variable and turned to lowercase
+    let question = document.getElementById("input").value.toLowerCase();
+
+    
+
+  //filter was used to return object which s inside the array  
+    const answer = chatbot.filter( item => item.input.includes(question))
+
+//created random number between 0 and 2
+    let randomNumber = Math.floor(Math.random()*3);
+   
+    
+//each radio button id assigned to a variable
+    let random =document.getElementById('random')
+    let shortest =document.getElementById('shortest')
+    let longest =document.getElementById('longest')
+   
+//conditional statement
+
+//if our array has any value of input it will return below condition else will give a massage
+    if(answer.length>0){
+
+                    if(shortest.checked === true){//for shortest answer
+                            document.getElementById("output").value +="you :  "+question+ '\n';
+                            document.getElementById("output").value +="bot :  "+ answer[0].output.sort((a, b) => a.length - b.length)[0]+ '\n' +'\n';
+
+                    }else if (longest.checked === true){//for longest answer
+                        document.getElementById("output").value +="you :  "+question+ '\n';
+                        document.getElementById("output").value +="bot :  "+answer[0].output.sort((a, b) => b.length - a.length)[answer.length-1]+ '\n' +'\n';
+                                             
+                    }else { //for random answer
+                        document.getElementById("output").value +="you :  "+question+ '\n';
+                        document.getElementById("output").value += "bot :  "+ answer[0].output.sort((a, b) => b.length - a.length)[randomNumber]+ '\n' + '\n';
+                    }
+
+        }else if (question.includes("dog")){//if input includes dog below function will work
+            
+                let image = new XMLHttpRequest(); //gives info from APIs
```

Comment:
I would abstract this into another function since this makes this function super long and doing too many things.. we want to make code as readable as possible